rs/types/types/src/messages.rs

```diff
@@ -801,35 +801,6 @@ mod tests {
         assert_matches!(response1, Ok(response1) if response == response1);
     }
 
-    #[test]
-    /// Allowing bincode::deserialize_from here since
-    /// 1. It's only being used in a test
-    /// 2. The deserialized_from is used on data that has just been serialized before the method.
-    #[allow(clippy::disallowed_methods)]
-    fn serialize_via_bincode_without_signature() {
```

Comment:
Why are we removing a test?

Reply:
I have no idea what the test is actually about. But it looks to me like it's testing an encode-decode roundtrip of an `HttpRequestEnvelope` with `sender_pubkey` and `sender_sig` set to `None`. The fact that it uses `bincode::deserialize_from()` instead of `bincode::deserialize()` appears (to my uninformed eyes) to be a side issue. I suppose the test would work just the same if switched to use `bincode::deserialize()`.

Reply:
I will temporary revert and dig a bit more why we have #[allow(clippy::disallowed_methods)] here